### PR TITLE
update URL to webpack configurator

### PR DESCRIPTION
**Pull request by jakoblind**
I have changed the URL to the webpack config tool so this PR reflects the new URL. I also added a better description.

### CODE REVIEW

File: src/content/configuration/index.mdx [hunk 1]
```diff
@@ -452,4 +452,4 @@ T> Want to rapidly generate webpack configuration file for your project requirem
 [Generate Custom Webpack Configuration](https://generatewebpackconfig.netlify.com/) is an interactive portal you can play around by selecting custom webpack configuration options tailored for your frontend project. It automatically generates
 a minimal webpack configuration based on your selection of loaders/plugins, etc.
 
-[Visual tool for creating webpack configs](https://webpack.jakoblind.no/) is an online configuration tool for creating webpack config where you can select any combination of features you need. It also generates a full example project based on your webpack configs.
+[createapp.dev - create a webpack config in your browser](https://createapp.dev/webpack) is an online configuration tool for creating webpack config where you can select any combination of features you need. It also generates a full example project based on your webpack configs that you can review in your browser and download.
```
Comment (EugeneHlushko) [inline src/content/configuration/index.mdx]: Suggesting small rephrase:
```suggestion
[createapp.dev - create a webpack config in your browser](https://createapp.dev/webpack) is an online tool for creating custom webpack configuration. It allows you to select various features that will be combined and added to resulting configuration file. Also, it generates an example project based on provided webpack configuration that you can review in your browser and download.
```

**Comment by EugeneHlushko** [comment]
By the way @jakoblind 
I find it great what you did there, i haven't used it yet but i hope that there will be time to try it out fully.

**Comment by EugeneHlushko** [comment]
Thanks!

**Comment by jakoblind** [comment]
Thanks @EugeneHlushko ! :+1:

**Comment by jakoblind** [comment]
Glad you liked the tool @EugeneHlushko ! I think especially devs who are new to webpack finds this tool useful for learning webpack. I was thinking maybe could put a link to it also somewhere in the webpack docs where new devs could see it, for example the [getting started page](https://webpack.js.org/guides/getting-started/). What do you think?

**Comment by EugeneHlushko** [comment]
Unfortunately that would open the flood gates for populating docs with 3rd party links, so lets keep it where such links belong

**Pull request merged**